### PR TITLE
dev-python/pipenv: fix import errors

### PR DESCRIPTION
**Pull request by oz123**
Bug: https://bugs.gentoo.org/717666
Signed-off-by: Oz N Tiram <oz.tiram@gmail.com>

### CODE REVIEW

**Review by juippis — APPROVED** [review]
I guess we'll want to revbump because this seemed to fix a runtime issues. I'll do it while merging, thanks!

**Comment by juippis** [comment]
Wait is the patch file even applied in the ebuild? Could you verify this is what you want to do? And add the patch to the ebuild to be applied, and then revbump to -r1 please.

**Comment by oz123** [comment]
@juippis , sorry, in the haste I forgot to include the build itself in the commit.

File: dev-python/pipenv/pipenv-2022.9.24-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+HOMEPAGE="https://github.com/pypa/pipenv https://pypi.org/project/pipenv/"
+SRC_URI="https://github.com/pypa/pipenv/archive/v${MY_PV}.tar.gz -> ${P}.gh.tar.gz"
+S="${WORKDIR}"/${PN}-${MY_PV}
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~riscv ~x86"
+
+PATCHES=(
+	"${FILESDIR}/${MY_PV}/pipenv-2022.9.24-inject-site-packages.patch"
```
Comment (juippis) [inline dev-python/pipenv/pipenv-2022.9.24-r1.ebuild:22]:
```
>>> Preparing source in /var/tmp/portage/dev-python/pipenv-2022.9.24-r1/work/pipenv-2022.9.24 ...
 * Applying pipenv-2022.9.24-inject-site-packages.patch ...
/var/tmp/portage/dev-python/pipenv-2022.9.24-r1/temp/environment: line 1969: /var/tmp/portage/dev-python/pipenv-2022.9.24-r1/files/2022.9.24/pipenv-2022.9.24-inject-site-packages.patch: No such file or directory
/var/tmp/portage/dev-python/pipenv-2022.9.24-r1/temp/environment: line 1972: /var/tmp/portage/dev-python/pipenv-2022.9.24-r1/files/2022.9.24/pipenv-2022.9.24-inject-site-packages.patch: No such file or directory
 [ !! ]
```
I'll fix, thanks!

**Pull request closed without merging**